packages/obconf.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Obconf < Package
+  description 'ObConf - Openbox configuration tool'
+  homepage 'https://openbox.org/wiki/ObConf:About'
+  version '2.0.4'
+  compatibility 'all'
+  source_url 'https://openbox.org/dist/obconf/obconf-2.0.4.tar.gz'
+  source_sha256 '71a3e5f4ee246a27421ba85044f09d449f8de22680944ece9c471cd46a9356b9' 
+
+  depends_on 'gtk2'
+  depends_on 'openbox'
```

Comment:
Should gtk2 be a dependency of openbox?

Reply:
Added `gtk2` and `gtk3` as `openbox` dependency

Reply:
If `gtk2` has been added to `openbox`, you can remove the dependency from this package, right?

Reply:
Done

Reply:
I still see `depends_on 'gtk2'` in this package.  That should move to `openbox`, correct?

Reply:
Never mind.  I didn't see the latest commit.